pkg/cmd/cli/cmd/deploy.go

```diff
@@ -227,7 +228,9 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	config.Status.LatestVersion++
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
-		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "--> Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "    Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
+		fmt.Fprintf(out, "    Use '%s scale --replicas=N dc/%s' to scale it accordingly.\n", o.baseCommandName, config.Name)
```

Comment:
> I don't know that this is the right place to suggest this. Why would I scale while a deployment is running?

Right, I added it mostly to replace "or even scale multiple deployments up at the same time." which I removed from the help text.